Failing to load sagui.config.js should not exit with status code 0

Errors when running tests are returning status 0, which means they work.

```
3.06s$ BROWSER='Mac Firefox 46.0' npm test
> @klarna/browser@0.0.1 test /home/travis/build/klarna/browser
> npm run test:unit
> @klarna/browser@0.0.1 test:unit /home/travis/build/klarna/browser
> cross-env NODE_ENV=test sagui test
  Sagui Error starting up
  Sagui Error: Cannot find module 'webpack'
    at Function.Module._resolveFilename (module.js:325:15)
    ( ... stacktrace ... )
The command "BROWSER='Mac Firefox 46.0' npm test" exited with 0.
```

--- a/src/cli/index.js
+++ b/src/cli/index.js
@@ -18,6 +18,7 @@ export default (argv = []) => {
 
     logError('Error starting up')
     logError(e.stack || e)
+    process.exit(1)
   }
 
   if (!argv.slice(2).length) program.outputHelp()

